### DISCUSSION TITLE
Unable to create sub-menu

### DISCUSSION THREAD

**Discussion by rirumel** [mmistakes/minimal-mistakes#2619] [Q&A] [answered]
I am a new user of Jekyl Minimal Mistakes theme. Recently, I am planning to develop a site where there will be sub-menu/ pages for a few of the parent menu options. But I am unable to achieve this.

It would be a great help if anyone could let me know whether it is possible, if yes what is the solution to achieve it? For example, if I click on “About” from the nav-bar, then the website will display the content of about.html as well as it will show the sub-pages that are available under the About page.

**Comment by iBug** [comment]
You can just link to the subpages in your "about" page.

**Comment by rirumel** [comment]
> You can just link to the subpages in your "about" page.

Something like this at the top of about page?

main:

  - title: "Goals"
    url: /goals.html

  - title: "Partners"
    url: /partners.html

**Comment by mmistakes** [comment, marked as answer]
The theme doesn't support multiple levels in the main navigation. You'd need to customize it if your intention was to have an About menu item in the header, and then have Goals and Partner links expand beneath that. See #1301

The theme supports a sidebar navigation which you could do this. Consult the documentation for how: https://mmistakes.github.io/minimal-mistakes/docs/layouts/#custom-sidebar-navigation-menu
